src/server/routes/api/document/index.js

```diff
@@ -1450,19 +1450,35 @@ const emailRelatedPaperAuthors = async doc => {
 
   const hasContact = paper => getContact(paper) != null;
 
+  const createEmptyDoc = async (authorName, authorEmail) => {
+    const provided = {
+      authorEmail,
```

Comment:
We need the `paperId` set here or else `postDoc` complains threre's not PubMed article found.

Reply:
I think hat `postDoc()`, if it doesn't already, should be allowed to fail in part or in whole.  There is no paper ID for this empty doc as per: 

- One-click editor link for related papers emails #925
- Post hoc specification of the paper in a factoid #901
- Post hoc titles etc #921
- Refactored RequestForm as a reusable component. #913

Reply:
Ah yes, empty doc my bad